adhocteam/Head-Start-TTADP
# Display reports

**Description of change**
Reports are displayed under "Activity Reports".  In this PR the table contains all reports in smart hub. The UI contains a "New report" button that allows to create a new report. If there are no reports, an empty row will be displayed in the table.

**How to test**
Local:
- pull changes
- yarn deps
- `yarn db:migrate`
- `yarn db:seed`
- `yarn start:local`
- click on "Activity Reports"
- verify the landing page has a table with "Activity reports"
- create a couple of reports
- verify they are displayed

Docker:
- same as above, but with the docker commands

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/5

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified

Note - "Fix Cucumber tests" branch was merged in to potentially avoid cucumber test failures.
